banquet/views.py

```diff
@@ -1058,6 +1101,64 @@ def invitation(request, year, token):
     )
 
 
+@permission_required("banquet.base")
+def send_invitation_button(request, year, banquet_pk, invitation_pk):
+    """Called when "Send invitation Mail" is pressed on the manage_invitation page."""
+
+    fair = get_object_or_404(Fair, year=year)
+    banquet = get_object_or_404(Banquet, fair=fair, pk=banquet_pk)
+    invitation = get_object_or_404(Invitation, banquet=banquet, pk=invitation_pk)
+
+    if invitation.user is None:
+        name = invitation.name
+        email = invitation.email_address
+        link = request.build_absolute_uri(
+            reverse(
+                "banquet_external_invitation",
+                kwargs={"token": invitation.token},
+            )
+        )
+    else:
+        name = invitation.user.get_full_name()
+        email = invitation.user.email
+        link = request.build_absolute_uri(
+            reverse(
+                "banquet_invitation",
+                kwargs={"year": year, "token": invitation.token},
+            )
+        )
+
+    send_invitation_mail(name, banquet.date, banquet.location, link, email)
+
+    return render(
+        request,
+        "banquet/invite_sent.html",
+        {
+            "fair": fair,
+        },
+    )
+
+
+def send_invitation_mail(name, date, location, link, email):
```

Comment:
Does this also contain a link to the invitation, so that you can press the link and easily get the QR code?

Reply:
![image](https://github.com/armada-ths/ais/assets/131291250/5b2d6de5-2f4e-4a33-b7ac-7251375fcc87)
Yes!